prisma/dbml/schema.dbml

```diff
@@ -0,0 +1,502 @@
+  }
+}
+
+Table RoleTerm {
+  id Int [pk, increment]
+  circleRole CirclesToRoles [not null]
+  circleId Int [not null]
+  roleId Int [not null]
+  holder Person [not null]
+  holderId String [not null]
+  start DateTime [default: `now()`, not null]
+  end DateTime
+}
+
+Table Ethic {
+  id Int [pk, increment]
+  shortName String [unique]
+  versions EthicVersion [not null]
+  activeId Int [unique, not null]
+  coreValue Boolean [not null, default: false]
+  responses EthicsToPersons [not null]
+
+  Note: 'A value that Moneyless Society collectively holds. Those that we require new members consent to are marked as `coreValue`s. Variations of the text are stored in each EthicVersion. After members propose values, other members will have the opportunity to review them and consent, change their ranking of variations, or provide commentary.'
+}
+
+Table EthicVersion {
+  id Int [pk, increment]
+  text String [unique, not null]
+  ethic Ethic [not null]
+  ethicId Int [not null]
+  submitter Person
+  submitterId String
+  createdAt DateTime [default: `now()`, not null]
+  updatedAt DateTime [not null]
+  responses EthicVersionsToPersons [not null]
+}
+
+Table EthicsToPersons {
+  ethic Ethic [not null]
+  ethicId Int [not null]
+  person Person [not null]
+  personId String [not null]
+  ranking Int[] [not null]
+  updatedAt DateTime [not null]
+
+  indexes {
+    (ethicId, personId) [pk]
+  }
+
+  Note: 'Versions of an ethical value\'s responses to different versions of a given ethical value are ranked in `versionRanking`.'
+}
+
+Table EthicVersionsToPersons {
+  ethicVersion EthicVersion [not null]
+  ethicVersionId Int [not null]
+  person Person [not null]
+  personId String [not null]
+  consent Boolean [not null, default: false]
+  commentary String
+
+  indexes {
+    (ethicVersionId, personId) [pk]
+  }
+
+  Note: 'Commentary and consent by an individual to a version of an ethical value.'
+}
+
+Table VolunteerQuestion {
+  id Int [pk, increment]
+  text String[] [not null]
+  responses VolunteerResponse [not null]
+  updatedAt DateTime [not null]
+  edits VolunteerQuestionEdits [not null]
+}
+
+Table VolunteerQuestionEdits {
+  question VolunteerQuestion [not null]
+  questionId Int [not null]
+  editor Person [not null]
+  editorId String [not null]
+  updatedAt DateTime [not null]
+
+  indexes {
+    (questionId, editorId) [pk]
+  }
+}
+
+Table VolunteerResponse {
+  question VolunteerQuestion [not null]
+  questionId Int [not null]
+  responder Person [not null]
+  responderId String [not null]
+  answer String [not null]
+  updatedAt DateTime [not null]
+  importedBy Person
+  importerId String
+  createdAt DateTime [default: `now()`, not null]
+  submittedAt DateTime
+
+  indexes {
+    (questionId, responderId) [pk]
+  }
+}
+
+Table OnboardingStep {
+  id Int [pk, increment]
+  name String [not null]
+  description String
+  volunteers OnboardingStepsToPersons [not null]
+}
+
+Table OnboardingStepsToPersons {
```

Comment:
Just a thought on naming conventions for table names. My preference to have something closer to natural language vs acronyms, also I would avoid adding the 'To' to things like circle to roles since a more natural way to say it would be circle roles.

Reply:
I can vibe with that. What informed my decision were the [Conventions for Implicit m-n Relation table names](https://www.prisma.io/docs/orm/prisma-schema/data-model/relations/many-to-many-relations#conventions-for-relation-tables-in-implicit-m-n-relations). This convention is not for explicit relations, but it made sense to me to match it more or less. If you could please check that out just to see my rationale, and you still feel the same, I'm game to change it.

Reply:
I guess since this is the prisma convention we should roll with that. Ty for the elucidating!